drivers/user_permission_report/app/views/user_permission_report/warehouse_reports/users/show.haml

```diff
@@ -0,0 +1,24 @@
+- content_for :modal_title, "Full Permissions for #{@user.name}"
+
+%h1= @user.name
+%h2= @user.email
+%h2 Roles
+.warehouse-reports__user-permission__profile-roles
+  - if @user.roles.any?
+    = @user.roles.pluck(:name).join(', ')
+  - else
+    No Roles
+%h2 Access Groups
+.warehouse-reports__user-permission__profile-access-groups
+  - if @user.access_groups.general.any?
+    = @user.access_groups.general.pluck(:name).join(', ')
+  - else
+    No Access Groups
+- group_associations.each do |key|
+  %h2= key.to_s.humanize.titleize 
+  .warehouse-reports__user-permission__profile-association
+    - associated_items = @user.access_groups.map { |ag| ag.public_send(key) }.flatten.uniq.map(&:name)
+    - if associated_items.any?
+      = associated_items.join(', ')
```

Comment:
These should be on their own lines

Reply:
That goes for all of the above loops (either as LI or with BR)

Reply:
✅